docs-md/app/etl.md

```diff
@@ -0,0 +1,608 @@
+```
+
+### Start the Postgres and MongoDB Debezium source connectors
+
+With all of our seed data in place, we can process it with ksqlDB. Connect to ksqlDB's server using its interactive CLI. Run the following from your host:
+
+```
+docker exec -it ksqldb-cli ksql http://ksqldb-server:8088
+```
+
+Before we issue more commands, instruct ksqlDB to start all queries from earliest point in each topic:
+
+```
+SET 'auto.offset.reset' = 'earliest';
+```
+
+Now we can connect ask Debezium to stream Postgres' changelog into Kafka. Invoke the following command in ksqlDB. This creates a Debezium source connector and writes all of its changes to Kafka topics:
+
+```sql
+CREATE SOURCE CONNECTOR customers_reader WITH (
+    'connector.class' = 'io.debezium.connector.postgresql.PostgresConnector',
+    'database.hostname' = 'postgres',
+    'database.port' = '5432',
+    'database.user' = 'postgres-user',
+    'database.password' = 'postgres-pw',
+    'database.dbname' = 'customers',
+    'database.server.name' = 'customers',
+    'database.history.kafka.bootstrap.servers' = 'broker:9092',
+    'database.history.kafka.topic' = 'customers',
+    'table.whitelist' = 'public.customers',
+    'transforms' = 'unwrap',
+    'transforms.unwrap.type' = 'io.debezium.transforms.ExtractNewRecordState',
+    'transforms.unwrap.drop.tombstones' = 'false',
+    'transforms.unwrap.delete.handling.mode' = 'rewrite'
+);
+```
+
+Notice that we specified an `unwrap` transform. By default, Debezium sends all events in an envelop that include many pieces of information about the change captured. Here, we only care about the value after it changed, so we instruct Kafka Connect to simply keep that information and discard the rest.
+
+Run another source connector to ingest the changes from MongoDB. We specify the same behavior for discarding the Debezium envelop:
+
+```sql
+CREATE SOURCE CONNECTOR logistics_reader WITH (
+    'connector.class' = 'io.debezium.connector.mongodb.MongoDbConnector',
+    'mongodb.hosts' = 'mongo:27017',
+    'mongodb.name' = 'my-replica-set',
+    'mongodb.authsource' = 'admin',
+    'mongodb.user' = 'dbz-user',
+    'mongodb.password' = 'dbz-pw',
+    'database.history.kafka.bootstrap.servers' = 'broker:9092',
+    'database.history.kafka.topic' = 'logistics',
+    'collection.whitelist' = 'logistics.*',
+    'transforms' = 'unwrap',
+    'transforms.unwrap.type' = 'io.debezium.connector.mongodb.transforms.ExtractNewDocumentState',
+    'transforms.unwrap.drop.tombstones' = 'false',
+    'transforms.unwrap.delete.handling.mode' = 'drop',
+    'transforms.unwrap.operation.header' = 'true'
+);
+```
+
+### Create the ksqlDB source streams
+
+For ksqlDB to be able to use the topics that Debezium created, we need to declare streams over it. Because we configured Kafka Connect with Schema Registry, we don't need to declare the schema of the data for the streams. It is simply inferred the schema that Debezium writes with.
```

Comment:
```suggestion
For ksqlDB to be able to use the topics that Debezium created, we need to declare streams over it. Because we configured Kafka Connect with Schema Registry, we don't need to declare the schema of the data for the streams. It simply infers the schema that Debezium writes with.
```